pkg/apis/pipeline/v1beta1/pipeline_defaults_test.go

```diff
@@ -0,0 +1,138 @@
+/*
+Copyright 2020 The Tekton Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1beta1_test
+
+import (
+	"context"
+	"testing"
+
+	"github.com/tektoncd/pipeline/test/diff"
+
+	"github.com/google/go-cmp/cmp"
+	"github.com/tektoncd/pipeline/pkg/apis/pipeline/v1beta1"
+)
+
+func TestPipeline_SetDefaults(t *testing.T) {
+	p := &v1beta1.Pipeline{}
+	want := &v1beta1.Pipeline{}
+	t.Run("set defaults for empty pipeline", func(t *testing.T) {
```

Comment:
since we've only got one test case we can drop the t.Run and just call the following lines directly

Reply:
Done